library/net_sockets.c

```diff
@@ -45,6 +45,8 @@
 #if (defined(_WIN32) || defined(_WIN32_WCE)) && !defined(EFIX64) && \
     !defined(EFI32)
 
+#define MBEDTLS_EINTR WSAEINTR
```

Comment:
Unfortunately `check-names.sh` rejects `MBEDTLS_EINTR` because it reserves the `MBEDTLS_` prefix for macros defines in `config.h`. I propose https://github.com/gilles-peskine-arm/mbedtls/commit/a1a16fed4fa8c0375dfd91c32002f69b893ee81a

Reply:
Note: not just `config.h`, any header file (as opposed to C files).

Reply:
@gilles-peskine-arm Thanks for spotting that and for providing a fix. I'm ok with you pushing it to this branch.

Reply:
I cherry-picked that commit onto your branch.